pkg/cloud/aws/services/ec2/instances.go

```diff
@@ -484,7 +482,15 @@ func (s *Service) CreateOrGetMachine(machine *actuators.MachineScope, bootstrapT
 		return instance, nil
 	}
 
-	return s.createInstance(machine, bootstrapToken)
+	instance, err = s.createInstance(machine, bootstrapToken)
+	if err != nil {
+		if !awserrors.IsFailedDependency(errors.Cause(err)) {
```

Comment:
What happens if it is a failed dependency?

Reply:
At the actuator/controller level it is requeued. This is to avoid spamming failure events for those cases.

Reply:
Thanks, that is quite helpful! Would you mind adding this info as a comment?